packages/shared/src/hooks/contexts.tsx

```diff
@@ -9,8 +9,10 @@ import type {
 } from '@clerk/types';
 import type { PropsWithChildren } from 'react';
 import React from 'react';
-import { SWRConfig } from 'swr';
 
+import { disableSWRDevtools } from './clerk-swr';
+disableSWRDevtools();
```

Comment:
Given that this is a top-level side effect already, I think we can simply call it inside once inside `clerk-swr.ts` directly. 

If we wanted to avoid top-level side-effects altogether, an alternative approach would be to create our own SWR wrapper and call `disableSWRDevtools` internally, eg:
```
import { useSWR as _useSWR } from 'swr';
export useSWR = () => {
    disableSWRDevtools();
    return _useSWR;
}
```

No real preference here :)

Reply:
We tried this, but was not working.

```ts
// file: clerk-swr.ts
  if (typeof window !== 'undefined') {
    // @ts-ignore
    window.__SWR_DEVTOOLS_USE__ = undefined;
  }


export * from 'swr';
export { default as useSWR } from 'swr';
export { default as useSWRInfinite } from 'swr/infinite';

```

Also swr setups the devtools once imported not once the hooks is called. Do you think your recommendation would still work with that in mind ?